Documentation/BuildInstructionsWindows.md

```diff
@@ -51,3 +51,22 @@ following command in an elevated PowerShell session: \
 You may have to reboot after enabling the WHPX feature.
 
 Afterwards you can start the VM with `Meta/serenity.sh run` as usual.
+
+## Note on Microsoft Store Ubuntu
```

Comment:
This information is not limited to microsoft store ubuntu, but to any distribution of Ubuntu 20.04 (or < 22.04). At the time we bumped to gcc 12, `ppa:ubuntu-toolchain-r/test` did not have gcc-12, but now it does. There used to be text in prerequisites that explained, this. Now that ubuntu-toolchain-r/test has the package, we should restore the previous text (with the correct version information). This note in windows could simply point out that you're likely not running the latest ubuntu when installing from windows store and to add the PPA by referencing the prereq's section.

https://github.com/SerenityOS/serenity/commit/bc2ebcadc0eb695605051c9d362c3f5dc184c63c#diff-2dc29685816e1b86c361a3184ccc2d05c4fdf910358f6718bb16321fd4a26ce6L16-L21